### PR TITLE
DOCSP-41883 Show binding redirects on upgrade page

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41883
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/csharp/DOCSP-41883-binding-redirects/upgrade/#version-2.28.0-potential-breaking-change
https://preview-mongodbjordansmith721.gatsbyjs.io/csharp/DOCSP-41883-binding-redirects/whats-new/#what-s-new-in-2.28

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -37,10 +37,15 @@ Learn what's new in:
 What's New in 2.28
 ------------------
 
+.. warning:: Potential Breaking Change in v2.28
+
+   - All {+driver-short+} components are strongly named. If your application
+     If your application has dependencies that reference multiple {+driver-short+}
+     versions, you must create binding redirects to manage those dependencies.
+     For more information, see :ref:`csharp-breaking-changes-2.28.0`.
```
Comment (mcmorisi) [inline source/whats-new.txt:45]:
```suggestion
   - All {+driver-short+} components are strongly named. If your application
     has dependencies that reference multiple {+driver-short+}
     versions, you must create binding redirects to manage those dependencies.
     For more information, see :ref:`csharp-breaking-changes-2.28.0`.
```

File: source/upgrade.txt [hunk 2]
```diff
@@ -50,6 +57,47 @@ changes between the current and upgrade versions. For example, if you
...
+
+     <configuration>
+         <runtime>
+             <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+                 <dependentAssembly>
+                     <assemblyIdentity name="MongoDB.Driver"
+                                       publicKeyToken="94992a530f44e321"
+                                       culture="neutral" />
+                     <bindingRedirect oldVersion="2.28.0.0"
+                                      newVersion="version number to use" />
```
Comment (mcmorisi) [inline source/upgrade.txt:79]: Line 93 has this written as "<version number to use>", would also surround this with the chevrons

File: source/upgrade.txt [hunk 3]
```diff
@@ -50,6 +57,47 @@ changes between the current and upgrade versions. For example, if you
...
+                 </dependentAssembly>
+                 <dependentAssembly>
+                     <assemblyIdentity name="MongoDB.Bson"
+                                       publicKeyToken="94992a530f44e321"
+                                       culture="neutral" />
+                     <bindingRedirect oldVersion="2.28.0.0"
+                                      newVersion="version number to use" />
```
Comment (mcmorisi) [inline source/upgrade.txt:86]: Same here

File: source/upgrade.txt [hunk 4]
```diff
@@ -50,6 +57,47 @@ changes between the current and upgrade versions. For example, if you
...
+                                       publicKeyToken="94992a530f44e321"
+                                       culture="neutral" />
+                     <bindingRedirect oldVersion="2.28.0.0"
+                                      newVersion="<version number to use>" />
+                 </dependentAssembly>
+             </assemblyBinding>
+         </runtime>
+     </configuration>
+
+
```
Comment (mcmorisi) [inline source/upgrade.txt:99]: Extra lines here
```suggestion
```

**Review by mcmorisi — APPROVED** [review]
LGTM with a few fixes!

File: source/whats-new.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/whats-new.txt]: 🤦‍♂️

**Pull request merged**